backend_new/src/services/application.service.ts

```diff
@@ -67,8 +74,11 @@ view.details = {
 @Injectable()
 export class ApplicationService {
   constructor(
+    @Inject(REQUEST) private req: ExpressRequest,
```

Comment:
From reading: https://docs.nestjs.com/fundamentals/injection-scopes

I'm concerned that this line may be causing re-instantiation of this service on each request which is something we wanted to get away from. 

I don't think we need the injection here either?

Reply:
was able to remove this